sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -182,7 +182,7 @@ public CloningBehavior getCloningBehavior() {
   }
 
   /**
-   * Calls {@link OldDoFn#startBundle} on the {@code OldDoFn} under test.
+   * Calls the {@link DoFn.StartBundle} on the {@link DoFn} under test.
```

Comment:
either "the StartBundle method" or "StartBundle"

I prefer the former, but up2u